app/src/features/admin/users/ActiveUsersList.tsx

```diff
@@ -264,6 +315,38 @@ const ActiveUsersList: React.FC<IActiveUsersListProps> = (props) => {
           />
         )}
       </Paper>
+
+      <EditDialog
+        dialogTitle={'Add System Users'}
+        open={openAddUserDialog}
+        dialogSaveButtonLabel={'Add'}
+        component={{
+          element: (
+            <AddSystemUsersForm
+              system_roles={
+                props.codes?.system_roles?.map((item) => {
+                  return { value: item.id, label: item.name };
+                }) || []
+              }
+            />
+          ),
+          initialValues: AddSystemUsersFormInitialValues,
+          validationSchema: AddSystemUsersFormYupSchema
+        }}
+        onCancel={() => setOpenAddUserDialog(false)}
+        onSave={(values) => {
+          handleAddSystemUsersSave(values);
```

Comment:
Something I didn't account for in the project participants:

If the save function fails, we still render the "3 users added message".
Easy fix is to move the `dialogContext.setSnackbar({...})` bit into the save function, so that if an error is thrown, it gets skipped.